src/cartesi-machine.lua

```diff
@@ -1656,9 +1548,6 @@ else
         assert(remote_address, "rollup requires --remote-address for snapshot/rollback")
         check_rollup_memory_range_config(config.rollup.tx_buffer, "tx-buffer")
         check_rollup_memory_range_config(config.rollup.rx_buffer, "rx-buffer")
-        check_rollup_memory_range_config(config.rollup.input_metadata, "input-metadata")
-        check_rollup_memory_range_config(config.rollup.voucher_hashes, "voucher-hashes")
```

Comment:
Do search all for `voucher_hashes` and `voucher-hashes`, there are still references for it in:
- `cartesi-machine.lua`
- `rollup-memory-range.lua`
- `htif-rollup.lua`
- `machine-bind.lua`

We could remove them right? Same for `notice_hashes` and `input_metadata`.